alcove/index/backend.py

```
@@ -12,52 +11,37 @@
 class ChromaBackend:
     """Vector backend backed by local ChromaDB."""
 
-    def __init__(self, embedder):
+    def __init__(self, embedder, collection_name: str | None = None):
```

Ruff (0.15.5)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)